components/inline-question-content.vue

```diff
@@ -1,6 +1,14 @@
 <template>
   <div class="content-wrap">
-    <h2 class="title">{{ question.title }}</h2>
+    <h2 class="title">
+      <img
```

Comment:
Wszystkie `img` muszą mieć atrybut `alt`. Jeśli obrazek jest tylko dekoracją, alt powinien być pusty -> to jest: `alt=""`